checkstyle-tester/diff.groovy

```diff
@@ -63,6 +63,8 @@ def getCliOptions(args) {
             + ' as a shorter version to prevent long paths. (optional, default is false)')
         m(longOpt: 'mode', args: 1, required: false, argName: 'mode', 'The mode of the tool:' \
             + ' \'diff\' or \'single\'. (optional, default is \'diff\')')
+        xm(longOpt: 'extraMvnOptions', args: 1, required: false, 'Extra arguements to pass to Maven' \
```

Comment:
There are a few maven command that these scripts run. Building and installing checkstyle, running checkstyle on the project, and building project-diff-util.
Can we add an extra clarification in the text (and maybe option name?) about which maven command we mean.

Reply:
Additionally add documentation on it.
https://github.com/checkstyle/contribution/tree/master/checkstyle-tester#command-line-arguments

Reply:
Done.